docs/dev/running-test.md

```diff
@@ -1,103 +1,73 @@
-# How to run Assisted-service subsystem tests
+# Running Assisted-service Subsystem Tests on Kubernetes
 
-There are two "flavors" of subsystem tests:
+## Overview
 
-* subsystem tests for the service deployed in REST-API mode
-* subsystem tests for the service deployed in kube-api mode
+This document details the steps required to run subsystem tests for the Assisted-service deployed in two modes:
+- REST-API mode
+- Kube-API mode
 
-Assisted-service subsystem tests require that you first deploy the
-assisted-service on a k8s cluster (e.g. minikube).
+## Deployment for Subsystem Tests
 
-This document will explain how you can easily deploy the service in preperation
-for subsystem tests, and also how to run the tests themselves.
+Assisted-service components will be deployed in a Kind cluster using the Podman provider. 
+`podman` and `kind` and `skipper` executables must be available in the $PATH.
+you can install kind by running:
 
-The subsystem tests themselves are located in the
-[subsystem](https://github.com/openshift/assisted-service/tree/master/subsystem)
-directory, but they are launched via `make` targets in the Makefile at the root
-of this repository.
-
-## Service deployment
-
-This section will show you how you can deploy the assisted installer to
-a minikube cluster in preperation for subsystem tests.
+```bash
+make install-kind-if-needed
+```
+which will install `kind` latest version if it is not present.
 
-### minikube
+## Deploying Components
 
-First we must prepare the minikube cluster -
+To deploy in REST-API mode, run:
 
 ```bash
-# Optionally delete the existing minikube cluster:
-# minikube delete
+make deploy-service-for-subsystem-test
```

Comment:
Can we clarify this must be run from the host? This target runs skipper

Reply:
I mentioned in the doc that it requires skipper

Reply:
Do you think we should add something else ?

Reply:
That's not what I meant, I think we should explicitly say this command needs to be executed directly from the host, without skipper.
I think it's already not very clear (for pre-existing targets) which targets are to be executed in skipper and which ones are to be executed from the host.

Reply:
So where you think we should specify it ?

Reply:
I added a comment in the makefile and it is specified in the doc